README.md

```diff
@@ -1,2 +1,3 @@
 # testing-notifications
```

Comment:
Mach mal hier n comment, an diesem komment

Reply:
mach mal so wie ich

Reply:
Testkommentar

Reply:
Testing

Reply:
This is a test comment